sdk/core/azure-core/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 - Renamed `NoRevoke` to `EnableCertificateRevocationListCheck` for `Azure::Core::Http::CurlTransportSSLOptions`.
 - Renamed `GetString()` to `ToString()` in `Azure::Core::DateTime`.
 - Renamed `GetUuidString()` tp `ToString()` in `Azure::Core::Uuid`.
+- Moved `NullBodyStream` to internal usage only. Not for public.
+- Removed `LimitBodyStream`.
```

Comment:
This is great! I was just about to do this :p

Reply:
I am adding github issues to the apiView review and assigning to me.
If there is anything else you are already working on, let me know to avoid duplicacion